juju/juju
# fix: remove gce firewall during teardown

On google cloud we don't remove the firewall created for the model, we only remove the
ports from the firewall. This fix ensures the firewall is cleaned up.

## QA steps

Bootstrap google.
Check for firewall rule created for the model/controller.
Destroy the controller.
Check the firewall rule is removed.

## Links

**Launchpad bug:** https://bugs.launchpad.net/juju/+bug/

**Jira card:** JUJU-
